inc/search.class.php

```diff
@@ -1205,8 +1337,10 @@ static function constructAdditionalSqlForMetacriteria($criteria = [],
     * @param boolean $onlycount If we just want to count results
     *
     * @return void
+    *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function constructData(array &$data, $onlycount = false) {
+   public function constructData(array &$data, $onlycount = false) {
```

Comment:
1. This method is used in some plugins (`formcreator` and `fusioninventory` for instance), change should be listed in changelog.

2. There is still 1 occurence of static usage of `Search::constructData()` in GLPI code.